bundle/Sail-Operator-ReadMe.md

```diff
@@ -0,0 +1,54 @@
+# About the Sail Operator
+
+The present implementation is based on the open source Istio project. The implementation pulls its code from the upstream Istio master with no changes to the codebase.
+
+# Prerequisites
+
+You have deployed a cluster on OpenShift Container Platform 4.13 or later.
+
+# Installing the Sail Operator
+
+1. Create the project where Istio is going to be deployed:  
+
+``` 
+$ oc new-project istio-system
+```
+
+2. And give it privileges (workaround while anyuid upstream PR is not merged)  
+
+```
+$ oc adm policy add-scc-to-group anyuid system:serviceaccounts:istio-system
+$ oc adm policy add-scc-to-group privileged system:serviceaccounts:istio-system
+```
+
+3. Navigate to the operator hub 
+
+4. Click Operator >> Operator Hub
+
+5. Search on Service Mesh. 
```

Comment:
Search for `sail` or `istio`

Reply:
I just said to search for sail, and confirmed it works. I like this instruction better because the only result is the sail operator.